### PR TITLE
Fix warning for missing Logger->messages

### PR DESCRIPTION
**Pull request by JeroenSormani**
Hi! 👋🏼

We were having a bunch of warnings added to the error log, going onto investigating it turned me to the value `SimpleCategoriesLogger->messages` being `null`. 

Took a bit of figuring out, but I believe this should fix it properly for your plugin, the warnings are all gone at least :-)

```
PHP Warning:  Invalid argument supplied for foreach() in /wp-content/plugins/simple-history/inc/SimpleHistory.php on line 1159
PHP Stack trace:
...
PHP   3. require_once() /wp-admin/admin-ajax.php:22
PHP   4. require_once() /wp-load.php:42
PHP   5. require_once() /wp-config.php:58
PHP   6. do_action($tag = *uninitialized*, $arg = *uninitialized*) /wp-settings.php:434
PHP   7. WP_Hook->do_action($args = *uninitialized*) /wp-includes/plugin.php:453
PHP   8. WP_Hook->apply_filters($value = *uninitialized*, $args = *uninitialized*) /wp-includes/class-wp-hook.php:310
PHP   9. SimpleHistory->load_loggers(*uninitialized*) /wp-includes/class-wp-hook.php:286
```

### CODE REVIEW

**Comment by bonny** [comment]
Thanks! Didn't know that the categories logger didn't have any search labels :)

**Comment by JeroenSormani** [comment]
Happy to assist :-)

Any known schedule for when the next release with this update will be published?
